src/components/Editor.jsx

```diff
@@ -58,13 +58,15 @@ class Editor extends React.Component {
   }
 
   componentWillUnmount() {
+    const {onComponentHidden, language} = this.props;
+    const {row, column} = this._editor.getCursorPosition();
+    onComponentHidden(language, row, column);
```

Comment:
Not sure if there is a better place for this action to be called, I would have ideally liked to call it from `EditorContainer.jsx`, where the hide function is called.